mobify/platform-scaffold
# Service worker in SDK

This PR switches the Merlins worker to use the central worker code in the SDK. Currently that is in PR https://github.com/mobify/progressive-web-sdk/pull/549/; until we merge, release, and update to the new release, this will fail on Circle

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/549/

## Changes
- Use the SDK worker 

## How to test-drive this PR
- Install the associated SDK branch
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that a new service worker is registered with version 0.2.0 (as printed in the console)
- Reload the home page twice
- See that all three caches are populated
- Go offline
- See that the home page loads again.
